# LiveParametersData Message

std_msgs/Header header

std_msgs/Bool valid
std_msgs/Float32 gyroBias
std_msgs/Float32 angleOffset
std_msgs/Float32 angleOffsetAverage
std_msgs/Float32 stiffnessFactor
std_msgs/Float32 steerRatio
std_msgs/Bool sensorValid
std_msgs/Float32 yawRate
std_msgs/Float32 posenetSpeed
std_msgs/Bool posenetValid



# struct LiveParametersData {
#   valid @0 :Bool;
#   gyroBias @1 :Float32;
#   angleOffset @2 :Float32;
#   angleOffsetAverage @3 :Float32;
#   stiffnessFactor @4 :Float32;
#   steerRatio @5 :Float32;
#   sensorValid @6 :Bool;
#   yawRate @7 :Float32;
#   posenetSpeed @8 :Float32;
#   posenetValid @9 :Bool;
# }
